api/spec/v1.yaml

```diff
@@ -767,6 +769,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/RuntimeTransactionList'
         <<: *common_error_responses
+  
+  /emerald/events:
+    get:
+      summary: Returns a list of Emerald events.
+      parameters:
+        - *limit
+        - *offset
+        - in: query
+          name: block
+          schema:
+            type: integer
+            format: int64
+          description: A filter on block round.
+          example: *runtime_block_round_1
+        - in: query
+          name: tx_index # TODO: Is it possible to get the tx_index from the oasis-sdk (currently only returns txhash)? Could make a second call to fetch the tx by its hash...
+          schema:
+            type: integer
+            format: int32
+          description: |
+            A filter on transaction index. The returned events all need to originate
+            from a transaction that appeared in `tx_index`-th position in the block.
+            It is invalid to specify this filter without also specifying a `block`.
+            Specifying `tx_index` and `round` is an alternative to specifying `tx_hash`;
+            either works to fetch events from a specific transaction.
+          example: 3
+        - in: query
+          name: tx_hash
+          schema:
+            type: string
+          description: |
+            A filter on the hash of the transaction that originated the events.
+            Specifying `tx_index` and `round` is an alternative to specifying `tx_hash`;
+            either works to fetch events from a specific transaction.
+          example: *tx_hash_1
+        - in: query
+          name: rel
+          schema:
+            type: string
+          description: |
+            A filter on related accounts. Every returned event will refer to
+            this account. For example, for a `Transfer` event, this will be the
+            the sender or the recipient of tokens.
+          example: *staking_address_1
+        - in: query
+          name: type
+          schema:
+            $ref: '#/components/schemas/ConsensusEventType'
+          description: A filter on the event type.
+          example: *runtime_event_type_1
```

Comment:
Nit:
You can delete this line and bake the example into RuntimeEventType.
Also, no need for the YAML references (`&` and `*`). Just list the example directly, it makes the YAML more readable. Since the example is a part of the type, we won't need to reference it anywhere else.